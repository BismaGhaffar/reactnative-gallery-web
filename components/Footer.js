import React from 'react'
import styled from 'styled-components'
import { Github, Mail, Minus } from 'react-feather'

const Footer = styled.section`
  position: absolute;
  left: 0;
  right: 0;
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 25px;
  margin-top: 100px;
  background-color: #fff;
  min-height: 100px;
  z-index: 1000;
  text-align: center;
  color: #333;
`

const Link = styled.a`
  color: #333;
  &:visited: {
    color: #333;
  }
`
function Foot() {
  return (
    <Footer>
      <Link
        href="https://github.com/ReactNativeGallery/reactnative-gallery-web"
        target="_blank"
        rel="noopener noreferrer"
      >
        <Github />
      </Link>
      <Minus style={{ paddingLeft: 15, paddingRight: 15 }} />
      <Link href="mailto:xcapetir+rng@gmail.com" target="_blank" rel="noopener noreferrer">
        <Mail />
      </Link>
    </Footer>
  )
}

export default Foot
