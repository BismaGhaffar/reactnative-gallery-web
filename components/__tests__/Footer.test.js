import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../Footer'

it('Footer can be created', () => {
  const comp = renderer.create(<Footer />)
  expect(comp).toBeDefined()
})

// it('<Footer /> toMatchSnapshot', () => {
//   const tree = renderer.create(<Footer />).toJSON()
//   expect(tree).toMatchSnapshot()
// })
