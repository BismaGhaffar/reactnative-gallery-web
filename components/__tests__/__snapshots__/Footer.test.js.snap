// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> toMatchSnapshot 1`] = `
<section
  className="Footer-s19zc472-0 cjLHqP"
>
  <a
    className="Footer__Link-s19zc472-1 kgBtyw"
    href="https://github.com/ReactNativeGallery/reactnative-gallery-web"
    rel="noopener noreferrer"
    target="_blank"
  >
    <svg
      fill="none"
      height="24"
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="2"
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
      />
    </svg>
  </a>
  <svg
    fill="none"
    height="24"
    stroke="currentColor"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    style={
      Object {
        "paddingLeft": 15,
        "paddingRight": 15,
      }
    }
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <line
      x1="5"
      x2="19"
      y1="12"
      y2="12"
    />
  </svg>
  <a
    className="Footer__Link-s19zc472-1 kgBtyw"
    href="mailto:xcapetir+rng@gmail.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    <svg
      fill="none"
      height="24"
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="2"
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z"
      />
      <polyline
        points="22,6 12,13 2,6"
      />
    </svg>
  </a>
</section>
`;
